test/MUXControlsTestApp/Utilities/ExtendedObservableCollection.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+
+namespace MUXControlsTestApp.Utilities
+{
+    class ExtendedObservableCollection<T> : ObservableCollection<T>
```

Comment:
Can you add some comments about why this class is created?

Reply:
Sure!